lib/griddler/sendgrid/adapter.rb

```diff
@@ -21,7 +21,7 @@ def normalize_params
             report: params[:spam_report],
             score: params[:spam_score],
           }
-
+          text: params[:text]&.to_s&.force_encoding(Encoding::UTF_8)
```

Comment:
Does this line have a spec I'm not seeing? Why is `.force_encoding(Encoding::UTF_8)` good enough here but line 82 below requires `filename.force_encoding('ISO-8859-1').encode('UTF-8')`?